rotkehlchen/chain/base/modules/hop/constants.py

```diff
@@ -19,3 +19,8 @@
         amm_wrapper=ZERO_ADDRESS,
     ),
 }
+
+REWARD_CONTRACTS: Final = {
+    string_to_evm_address('0x7aC115536FE3A185100B2c4DE4cb328bf3A58Ba6'),  # HOP (USDC.e)
+    string_to_evm_address('0x12e59C59D282D2C00f3166915BED6DC2F5e2B5C7'),  # HOP (ETH)
```

Comment:
I'm not sure if you told me. Is there any registry for those or api to query?

Reply:
There were none, I had checked during the bridge decode PR.